docs/pages/dsDisplay/ds-display_L3_Test-Procedure.md

```diff
@@ -0,0 +1,305 @@
+
+#### Test Suite Configuration
+
+Update the execute command according to the device path where `HAL` binaries are copied and Update the test suite for each level test case
+
+- Example configuration file: [dsDisplay_test_suite.yml](../../../host/tests/dsClasses/dsDisplay_test_suite.yml).
+
+## Run Test Cases
+
+Once the environment is set up, you can execute the test cases with the following command
+
+```bash
+python <TestCaseName.py> --config </PATH>/ut/host/tests/configs/example_rack_config.yml --deviceConfig </PATH>/ut/host/tests/configs/deviceConfig.yml
+```
+
+## Test Case Procedure
+
+### dsDisplay_test01_VerifyDisplayConnectCallBackTest.py
+
+**Overview:**
+
+This test verifies display event callbacks by connecting and disconnecting a display device on each supported video port, ensuring that appropriate callbacks are triggered for display connection, disconnection, Rx Sense ON, Rx Sense OFF, and HDCP protocol version change events.
+
+**Platform Supported:**
+
+- Source
+- Sink
+
+**User Input Required:**
+
+Yes: User input is required to manually change the display status (connect/disconnect) during the test execution, with the prompt question `Change the {port} status to {event} and press Enter:` (This will be automated later).
+
+**Acceptance Criteria:**
+
+The test will pass if each display event triggers the expected callback for every supported display port.
+
+**Expected Results:**
+
+The test will raise events for connecting and disconnecting the display and verify that the expected callbacks are received for each event. The test will pass if all expected callbacks are detected; otherwise, it will fail.
+
+**Test Steps:**
+
+- Run the Python file `dsDisplay_test01_VerifyDisplayConnectCallBackTest.py` with the appropriate configuration:
+
+```bash
+dsDisplay_test01_VerifyDisplayConnectCallBackTest.py --config /host/tests/configs/example_rack_config.yml --deviceConfig /host/tests/configs/deviceConfig.yml
+```
+
+- The test will download all required artifacts, copy them to the target directory, and start execution.
+
+- Callback Event Verification:
+
+  - Prompting the user to change display status for each supported port (connect/disconnect)
+  - Capture and verify the device's callback status to confirm receipt of the expected event callback.
+  - For each event, verify the following:
+    - dsDISPLAY_EVENT_CONNECTED
+    - dsDISPLAY_EVENT_DISCONNECTED
+    - dsDISPLAY_RXSENSE_ON
+    - dsDISPLAY_RXSENSE_OFF
+    - dsDISPLAY_HDCPPROTOCOL_CHANGE
+
+- Completion and results:
+
+  After verifying all callback statuses, the test will conclude and display the final result (PASS/FAIL).
+
+### dsDisplay_test02_TestVerifyDisplayEdid.py
+
+**Overview:**
+
+This test retrieves and verifies `EDID` by connecting various pre-defined displays to each supported video port. It then compares the retrieved `EDID` data against expected values to ensure accurate display identification.
```

Comment:
are you connecting the various displays for a source device?  Let's define the platforms (source or sink) for this module clearly.